### PR TITLE
Last blocks page to mark which chunks were skipped

### PR DESCRIPTION
**Pull request by mm-near**

### CODE REVIEW

File: chain/jsonrpc/res/last_blocks.html [hunk 1]
```diff
@@ -99,6 +100,21 @@
...
                     });
+
+                    for (let i = 0; i <= max_shard; i += 1) {
+                        let chunk_span = "chunk_" + i;
+                        let entries = $("." + chunk_span);
+                        let previous_hash = ""
+                        for (let j = entries.length - 1; j >= 0; j -= 1) {
+                            let new_hash = entries[j].attributes['hashvalue'].value;
+                            if (previous_hash == new_hash) {
+                                entries[j].style['background-color'] = 'grey';
```
Comment (mina86) [inline chain/jsonrpc/res/last_blocks.html:111]: Could we do it through a class instead?

File: chain/jsonrpc/res/last_blocks.html [hunk 2]
```diff
@@ -32,9 +32,10 @@
     </style>
     <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.5.1/jquery.min.js"></script>
     <script>
-        function createHashElement(hashValue, creator) {
+        function createHashElement(hashValue, creator, className) {
             var element = $('<span>');
             element.addClass('hashElem');
+            element.addClass(className);
             element.attr("hashValue", hashValue);
```
Comment (mina86) [inline chain/jsonrpc/res/last_blocks.html:39]: `"hashvalue"` (or alternatively use `"hashValue"` where you read the value).

File: chain/jsonrpc/res/last_blocks.html [hunk 3]
```diff
@@ -87,8 +88,8 @@
                         for (let i = 0; i < (max_shard + 1) * 3; i += 1) {
                             shards_cells.push($('<td>'));
                         }
-                        element.chunks.forEach(chunk => {
-                            shards_cells[chunk.shard_id * 3].append(createHashElement(chunk.chunk_hash, chunk.chunk_producer));
+                        element.chunks.forEach((chunk, index) => {
+                            shards_cells[chunk.shard_id * 3].append(createHashElement(chunk.chunk_hash, chunk.chunk_producer, "chunk_" + index));
```
Comment (mina86) [inline chain/jsonrpc/res/last_blocks.html:92]: Why not just have `let chunks_by_index = {};` outside of this for loop and then:

```javascript
let element = createHashElement(chunk.chunk_hash, chunk.chunk_producer)
(chunks_by_shard[index] = chunks_by_shard[index] || []).push(element);
shards_cells[chunk.shard_id * 3].append(element);
```

Using DOM seems unnecessary.

**Review by mina86 — APPROVED** [review]

File: chain/jsonrpc/res/last_blocks.html (hunk 3 above)
Comment (mm-near) [reply chain/jsonrpc/res/last_blocks.html]: done

File: chain/jsonrpc/res/last_blocks.html (hunk 2 above)
Comment (mm-near) [reply chain/jsonrpc/res/last_blocks.html]: done

**Pull request merged**